packages/scaffolding-cli/templates/src/csr/package.json

```diff
@@ -35,5 +89,60 @@
       "last 1 firefox version",
       "last 1 safari version"
```

Comment:
Inherit?

Reply:
So when I do the eslint story this will be tidied up.